src/core/reducers/addons.js

```diff
@@ -194,7 +194,7 @@ export function createInternalAddon(apiAddon: ExternalAddonType): AddonType {
     // TODO: remove this if possible. This is used by core/installAddon
     // and DiscoPane components which do camel case conversions for
     // some historic reason.
-    iconUrl: apiAddon.icon_url,
+  
```

Comment:
Also please check https://travis-ci.org/mozilla/addons-frontend/jobs/420119929#L515 , you have trailing space here.
Please make sure to run `yarn prettier-dev` before commit

Reply:
okk @championshuttler !